app/src/js/utils/format.js

```diff
@@ -405,221 +406,114 @@ export const collectionHrefFromNameVersion = ({ name, version } = {}) => {
   return `/collections/collection/${name}/${encodeURIComponent(version)}`;
 };
 
+/* Modal Text */
 export const enableText = (name) => [
-  'You have submitted a request to enable the following rule:',
-  <br />,
-  <br />,
-  <b>{name}</b>,
-  <br />,
-  <br />,
-  'Are you sure you want to enable this rule?',
+  <p>You have submitted a request to enable the following rule:</p>,
```

Comment:
So Danielle just set this the other way in the 2511 PR 
https://github.com/nasa/cumulus-dashboard/pull/983/files#diff-9d3605f4207afea46acf4d2ceb7030f42492c4480327281d8cda7bfbaec5d131R408
I don't have a preference but we should decide which format to use.

Reply:
@kkelly51 Danielle didn't actually write the code for it (that was done here https://bugs.earthdata.nasa.gov/browse/CUMULUS-2358) but she ran the formatter when she had to merge develop in. I'm just cleaning it up so we don't have to use so many breaks (<br>) and just control the spacing and styling through CSS.